pdf-toolbox/src/test/java/com/lowagie/examples/objects/bookmarks/PageLabels.java

```diff
@@ -65,5 +66,19 @@ public static void main(String[] args) {
 
         // step 5: we close the document
         document.close();
+
+        try (PdfReader reader = new PdfReader("PageLabels.pdf")) {
+            final String[] pageLabels = PdfPageLabels.getPageLabels(reader);
+            for (int i = 0; i < pageLabels.length; i++) {
+                System.out.println("pageLabels = " + pageLabels[i]);
```

Comment:
No entries of any `System` stream allowed. If you want to log smth, than use loggers. However, what is the purpose of these particular log messages?

Reply:
I'll transform this an assert as it is a test class